Library/Homebrew/dev-cmd/test-bot.rb

```diff
@@ -920,7 +920,6 @@ def sanitize_ARGV_and_ENV
     end
 
     ENV["HOMEBREW_DEVELOPER"] = "1"
-    ENV["HOMEBREW_NO_AUTO_UPDATE"] = "1"
```

Comment:
Thanks for bearing with me! :bow: I misread the code and thought `HOMEBREW_NO_AUTO_UPDATE` would only be exported if a `brew install` or `brew upgrade` command runs, but I see now that it's always set. Sorry for the noise!